torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -1321,6 +1321,23 @@ void LLVMCodeGenImpl::emitIsNan(const Intrinsics* v) {
   }
 }
 
+static bool wantSleef(const std::string& name) {
+  // Using sleef on these ops is slower than libm.
+  static std::unordered_set<std::string> noSleef = {
+      "sqrt",
+      "ceil",
+      "trunc",
+      "fabs",
+      "floor",
+      "sqrtf",
+      "ceilf",
+      "truncf",
+      "fabsf",
+      "floorf",
+  };
+  return noSleef.find(name) == noSleef.end();
```

Comment:
nit: i find `noSleef.count(name) == 0` to be a little easier to parse personally (not worth updating this PR just for this comment ofc)

Reply:
Haha, I do too generally, I have no idea what possessed me to do `find == end`.